src/compiler/types.ts

```diff
@@ -2719,6 +2719,7 @@ namespace ts {
         inlineSources?: boolean;
         isolatedModules?: boolean;
         jsx?: JsxEmit;
+        jsxFactory?: string;

```

Comment:
This is somewhat similar to what we do for `externalHelpersModuleName` on `SourceFile`.